specs/electra/beacon-chain.md

```diff
@@ -648,8 +656,14 @@ def queue_entire_balance_and_reset_validator(state: BeaconState, index: Validato
     validator = state.validators[index]
     validator.effective_balance = 0
     validator.activation_eligibility_epoch = FAR_FUTURE_EPOCH
-    state.pending_balance_deposits.append(
-        PendingBalanceDeposit(index=index, amount=balance)
+    state.pending_deposits.append(
+        PendingDeposit(
+            epoch=get_current_epoch(state),
+            pubkey=validator.pubkey,
+            withdrawal_credentials=validator.withdrawal_credentials,
+            amount=balance,
+            signature=BLSSignature(),
```

Comment:
why do we need to keep the `signature`? we can verify that before adding to this list right?